canonical/canonical.com
# Data bubble navigation

## Done

Navigation for data bubble /data
Added links under logo
Update marketo IDs according to:
- https://docs.google.com/document/d/1bfo97XdFdKlzrMcMoRxiTdzt_bPVndx6j55IPsapuAE/edit?disco=AAAA6wvGRUc&usp_dm=true
- https://docs.google.com/document/d/1nVdoKoPrXw8lsvHEtMichlPNfxYU9Uj1zmKowcUXQ_w/edit?disco=AAAA35h92vk&usp_dm=true

## QA

- Go to https://canonical-com-1056.demos.haus/data and check the navigation
- Run through the following [QA steps](https://discourse.canonical.com/t/qa-steps/152)


## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-6449
